app/screens/code/code.js

```diff
@@ -70,21 +75,28 @@ export default class Code extends React.PureComponent {
         let textComponent;
         if (Platform.OS === 'ios') {
             textComponent = (
-                <TextInput
-                    editable={false}
-                    multiline={true}
-                    value={this.props.content}
-                    style={[style.codeText]}
-                />
+                <SyntaxHighlighter
+                    language={this.props.language}
+                    style={getHighlightStyleFromTheme(this.props.theme)}
+                    highlighter={'hljs'}
+                    CodeTag={TextInput}
+                    codeTagProps={{editable: false, multiline: true, style: {...style.codeText, ...this.props.textStyle}}}
+                >
+                    {this.props.content}
+                </SyntaxHighlighter>
             );
         } else {
             textComponent = (
-                <Text
-                    selectable={true}
-                    style={style.codeText}
+                <SyntaxHighlighter
+                    language={this.props.language}
+                    style={getHighlightStyleFromTheme(this.props.theme)}
+                    highlighter={'hljs'}
+                    customStyle={{...style.codeText, ...this.props.textStyle}}
```

Comment:
This line is constructing a new object that gets passed in as the `customStyle` prop each time. Perhaps this is causing the `SyntaxHighlighter` to re-render too often, causing the lag? If we don't create a new object each time, that could possibly help here.

The same thing applies to the line below for `codeTagProps`.

Reply:
I'll investigate. Thanks!